README.md

```diff
@@ -1,12 +1,11 @@
-[![CircleCI](https://circleci.com/gh/google/pybadges.svg?style=svg)](https://circleci.com/gh/google/pybadges)
-![pypi](https://img.shields.io/pypi/v/pybadges.svg)
-![versions](https://img.shields.io/pypi/pyversions/pybadges.svg)
-   
 # pybadges
 
 pybadges is a Python library and command line tool that allows you to create
 Github-style badges as SVG images. For example:
 
+[![CircleCI](https://circleci.com/gh/google/pybadges.svg?style=svg)](https://circleci.com/gh/google/pybadges)
```

Comment:
The CircleCI, etc. badges aren't generated by `pybadges` though.

Reply:
Sorry! Feel free to close this pull request in that case.